src/commands/starter/starter.command.ts

```diff
@@ -0,0 +1,121 @@
+import { Message, RichEmbed, User } from 'discord.js';
+import { prefixedCommandRuleTemplate , defaultEmbedColor } from '../../config';
+import {
+    Command,
+    CommandClass,
+    detectStaff
+  } from '../../shared';
+
+@Command({
+    matches: ['starter', 'starterpack', 'starterkit'],
+    ...prefixedCommandRuleTemplate
+})
+
+    /**
+     * @param msg
+     * @param args
+     */
+
+export class StarterCommand implements CommandClass {
+    action(msg: Message, args: string[]) {
+        let whoTag;
```

Comment:
Let's do:
```
let whoTag = msg.author;
```
and then if Tophie accepts the pinging users, redefine `whoTag` there, so it's all in one piece, ya know?